src/applications/ezr/containers/App.jsx

```diff
@@ -24,15 +26,33 @@ const App = props => {
     user,
   } = props;
   const { veteranFullName } = formData;
-  const { loading: isLoadingFeatures, isProdEnabled, isSigiEnabled } = features;
+  const {
+    loading: isLoadingFeatures,
+    isProdEnabled,
```

Comment:
Why is this logic still in place? There should be a ticket created to remove it, as it's been in prod for over a year and isn't going anywhere.